### DISCUSSION TITLE
📣 Suggest chart type

### DISCUSSION THREAD

**Discussion by jung-han** [nhn/tui.chart#543] [Ideas]
# Suggest chart type

If you have a chart you would like to make or a chart you would like to make, please suggest us!

Before suggesting

1. Enter the site and check if the chart exists
2. Please upload the chart name, information, and specific image.

I will develop it after review!

⚠️  If you want to develop it yourself, please wait a little bit after the proposal and until we finish the design.

### Currently planned chart

* [ ] Radiar bar chart(`Work in Progress`)
* [ ] Gauge chart
* [ ] Candlestick chart

**Comment by vbelyakov** [comment]
Suggestion of two variants of radar chart type is here: https://github.com/nhn/tui.chart/issues/542.

The first type is similar to line chart where all series start from point (0,0). 
Use case example: there multiple cells in a field of view, Each cell has an initial position, i.e. (0, 0) and moves in time. The chart shows trajectories of all cells starting from each cell relative (0, 0) position.

The second chart is basically the same as the first one but with just one data point for each cell representing a Euclidean distance travelled by cell from initial position of (0,0).

**Reply by vbelyakov** [reply]
I wonder what would be a realistic ETA for charts in this post made available? Thanks.

**Reply by jung-han** [reply]
Currently, I'm preparing to develop a map chart. For this chart, I think we need to prioritize after that.

**Comment by rajhen96** [comment]
Suggestion for developing Gantt chart in Toast UI.The chart will be similar to bar chart but when we comes time range series, the Gantt structure will give precise data and give good visualization.

Example : Chart with Turn around time will use this charts

Chart Name: Gantt Chart or TimeRange Chart

Link: https://www.teamgantt.com/free-gantt-chart-excel-template
